ui/src/common/components/CitationsPapersLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoginOutlined } from '@ant-design/icons';
+
+import { Link } from 'react-router-dom';
+import { LITERATURE } from '../routes';
+import IconText from './IconText';
+import FormattedNumber from './FormattedNumber';
+import pluralizeUnlessSingle from '../utils';
+
+const CitationsPapersLink = ({ recordId, itemCount, displayedName }) => (
+  <Link to={`${LITERATURE}?q=refersto:recid:${recordId}`}>
```

Comment:
this link will work only for `citations` but it's used for other stuff, if you look at the snapshots from the tests, we need different queries.